### ISSUE TITLE
will you open source the training code?

### ISSUE THREAD

**Issue by BillCLYang** [XuanhaoLiu/MoGE#3]
thanks for your great job. i met overfitting issues when reproducing the training process according to the paper, for seed dataset. looking forward to your open-source training code.

**Comment by XuanhaoLiu** [comment]
Thanks for your interests of our work!

Overfitting is a common things in EEG-based emotion recognition field. You can search the hyperparameters and find the best accuracies.

**Comment by BillCLYang** [comment]
@XuanhaoLiu hi,  is gcn operating in d/hidden dimension? we notice that rearrange() is applied twice in forward func of class Moe_GCN_Transformer_unit. it seems gcn is operating in EEG channel dimension.

![Image](https://github.com/user-attachments/assets/29bd983d-8b10-41c8-83bb-4a04e4aa3e29)

**Comment by XuanhaoLiu** [comment]
Yes，GCN is designed to work on EEG channel dimension. Thanks for your interest!
